.gitignore

```diff
@@ -1,27 +1,29 @@
-# Docker project generated files to ignore
-#  if you want to ignore files created by your editor/tools,
-#  please consider a global .gitignore https://help.github.com/articles/ignoring-files
+# If you want to ignore files created by your editor/tools, please consider a
+# [global .gitignore](https://help.github.com/articles/ignoring-files).
+
 *.exe
 *.exe~
 *.gz
 *.orig
-test.main
 .*.swp
 .DS_Store
```

Comment:
Do we need a `thumbs.db` to help people working on Windows?

Reply:
Good call.